hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OzoneManager.java

```diff
@@ -2956,16 +2956,16 @@ public String getRatisRoles() {
         leaderId = omRatisServer.getLeader();
         if (leaderId == null) {
           LOG.error("No leader found");
-          return "Exception: Not a leader";
+          return Collections.emptyList();
         }
         serviceList = getServiceList();
       } catch (IOException e) {
         LOG.error("IO-Exception Occurred", e);
-        return "Exception: " + e;
+        return Collections.emptyList();
       }
       return OmUtils.format(serviceList, port, leaderId.getId().toString());
     } else {
-      return "Ratis-Disabled";
+      return Collections.emptyList();
```

Comment:
In case of standalone, i.e. ratis disabled, how the info will be shown? as list is empty
can see node info showing "CANDIDATE", do that will be enough?